src/back/server.py

```diff
@@ -150,13 +155,16 @@ def on_track(track):
         log_info("Track %s received", track.kind)
 
         if track.kind == "video":
-            global source_video
+            global source_video, broadcaster_active
             source_video = track
+            broadcaster_active = True
 
         @track.on("ended")
         async def on_ended():
             log_info("Track %s ended", track.kind)
             _transformed_tracks.clear()
+            global broadcaster_active
+            broadcaster_active = False
```

Comment:
The broadcaster_active doesn't get set back to False after the original broadcaster stops streaming